src/MoBi.Core/Services/IPKSimStarter.cs

```diff
@@ -11,5 +11,6 @@ public interface IPKSimStarter
       IBuildingBlock CreateProfileExpression(ExpressionType expressionType);
       IBuildingBlock CreateIndividual();
       IReadOnlyList<ExpressionParameterValueUpdate> UpdateExpressionProfileFromDatabase(ExpressionProfileBuildingBlock expressionProfile);
+      SimulationTransfer RecreateSimulationTransfer(string snapshot);
```

Comment:
I'd rename this as snapshotSerialization or sthg liek this because it's hard to understrand what this is otherwise